app/Models/Participant.php

```diff
@@ -11,7 +11,7 @@ class Participant extends Model
     use HasFactory, SoftDeletes;
 
     // RELATION
-    public function events()
+    public function joinEvent()
```

Comment:
I suggest adding `s` to the function name since it will return multiple records right?

Reply:
Yes. I'll change